HHS/Head-Start-TTADP
# [TTAHUB-1782] Add new permissions, feature flag, and AR training page

## Description of change

This PR starts to lay the foundation for the new 'Training reports page' frontend. It adds a feature flag for training reports as well as read and write permission for the new report type.

**Note:** If your user has ADMIN you will see the Training reports NAV regardless of feature flag settings.

**New Flag/Permissions:**
![image](https://github.com/HHS/Head-Start-TTADP/assets/84350609/b71887da-706e-45f6-bd23-38d1dfd583f0)

**New Page:**
![image](https://github.com/HHS/Head-Start-TTADP/assets/84350609/12c76d53-cdf2-4ed7-b4ec-12d0b05bd75b)


## How to test

1. Review the code
2. For a non-admin user make sure you can set/change the feature flag for training reports.
3. A non-admin user without the feature flag should NOT be able to go to the new page.
4. A non-admin user with the feature flag should be able to visit the page.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1782


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
